src/components/StackedChart/index.js

```diff
@@ -0,0 +1,306 @@
+import dayjs from "dayjs";
+import PropTypes from "prop-types";
+import React, { useCallback, useEffect, useState } from "react";
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Bar,
+  ComposedChart,
+} from "recharts";
+
+import { SupportedNetwork } from "../../constants";
+import { formattedNum, formattedPercent } from "../../utils";
+import Header from "./Header";
+import CrosshairTooltip from "./CrosshairTooltip";
+
+const TIME_FILTER_OPTIONS = {
+  MONTH_1: "1M",
+  MONTH_3: "3M",
+  YEAR: "1Y",
+  MAX: "MAX",
+};
```

Comment:
```suggestion
const TIME_FILTER_OPTIONS = {
  MONTH_1: "1M",
  MONTH_3: "3M",
  YEAR: "1Y"
};
```
Can max it to one year. 
 - One year data itself chart is slow. We are also getting max 1 year response from API
 - If we plot more than one year data in small space, users cant get that much information from it. 
 
 For better UX its better to limit data to 1year.